src/app/[lang]/verify-request/page.tsx

```diff
@@ -0,0 +1,53 @@
+import { getDictionary } from "@/get-dictionary";
+import { type Locale } from "@/i18n-config";
+import { VerifyTokenInput } from "./verify-token-input";
+import { Card, CardHeader, CardContent } from "@/components/ui/card";
+import { ArrowLeft } from "lucide-react";
+import Link from "next/link";
+import { type Metadata } from "next";
+import { ResendTimer } from "./resend-timer";
+
+export async function generateMetadata({ params: { lang } }: { params: { lang: Locale } }): Promise<Metadata> {
+    const dictionary = await getDictionary(lang);
+    return {
+        title: dictionary.auth.verifyRequest.title + " | FixMyTown",
+        description: dictionary.auth.verifyRequest.description,
+    };
+}
+
+
+export default async function VerifyRequestPage({
+    params: { lang },
+}: {
+    params: { lang: Locale };
+}) {
+    const dictionary = await getDictionary(lang);
+
+    return (
+        <div className="min-h-screen flex flex-col items-center justify-center bg-gradient-to-br from-primary via-accent to-secondary p-4">
```

Comment:
The text colour is implemented incorrectly for this for cases with light mode on. (Purple colour on white ground for light mode)